lib/logstash/inputs/opensearch.rb

```diff
@@ -347,7 +349,7 @@ def hosts_default?(hosts)
   end
 
   def setup_ssl
-    @ssl && @ca_file ? { :ssl  => true, :ca_file => @ca_file } : {}
```

Comment:
We shouldn’t remove ca_file. Create new condition if ssl true and ca file is false then add this